examples/README.md

```diff
@@ -0,0 +1,39 @@
+# CI Pipeline Examples
+
+## Jenkins Pipeline
+
+### Requirements
+
+#### Credentials
+
+* Github
+
+Github credentials will need to be added to Jenkins Global Credential Manager as the ID of 'github'. This should be the username and a personal access token added with all 'repo' and child object permissions.
```

Comment:
```suggestion
Github credentials will need to be added to Jenkins Global Credential Manager as the ID of 'github'. This should be the username and a personal access token added with all 'repo' and child object permissions. Personal access tokens can be created at https://github.com/settings/tokens.
```

Reply:
Allowing access to all repo permissions seems overly broad. Does this truly need to read and write security events, for example?

![image](https://user-images.githubusercontent.com/5713754/119425480-790ec880-bcd5-11eb-899d-6f7465c262c0.png)

Reply:
Everywhere I look defines all children of repo permission. Let me test some things.

Reply:
yeah.. I cant get it to work without the entire group.

Reply:
So be it. At least we tried and documented the results!